netbox/utilities/templatetags/helpers.py

```diff
@@ -230,14 +230,22 @@ def querystring(request, **kwargs):
 
 
 @register.inclusion_tag('utilities/templatetags/utilization_graph.html')
-def utilization_graph(utilization, warning_threshold=75, danger_threshold=90):
+def utilization_graph(data_input, warning_threshold=75, danger_threshold=90):
     """
     Display a horizontal bar graph indicating a percentage of utilization.
     """
+    # Check for possible division by zero error
+    if data_input[1] == 0:
+        utilization = 0
+    else:
+        utilization = int(float(data_input[0]) / data_input[1] * 100)
+
     return {
         'utilization': utilization,
         'warning_threshold': warning_threshold,
         'danger_threshold': danger_threshold,
+        'utilization_count': data_input[0],
+        'num_available': data_input[1],
```

Comment:
Should `num_available` indicate the _total_ number of objects, or the remaining number after accounting for `utilization_count`? It's confusing because the tooltip will currently show e.g. "Available: 254" for a /24 IPv4 prefix regardless of its `utilization_count`.

Reply:
Yes, this is off as I look at it here again.

Should we expand this to 3 items, including with an example of Prefix from the screenshot above:

| Var Name | Represents | Example | Label |
| ---------- | ----------- | -------- | ------ |
| utilization_count | Number of used address | 21 | Used |
| *No variable*, handle with math | Number of available to be used after utilization accounted for | 233 (254-21) | Available |
| total_possible (rename from num_available) | Total addresses within the prefix | 254 | Total Count |

Reply:
This only needs to convey the total possible and the number consumed/created.